src/Symfony/Component/HttpKernel/Tests/DependencyInjection/AddCacheClearerPassTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\DependencyInjection;
+
+use Symfony\Bridge\PhpUnit\ErrorAssert;
+use Symfony\Component\HttpKernel\DependencyInjection\AddCacheClearerPass;
+
+class AddCacheClearerPassTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @group legacy
+     * @dataProvider nullProvider
```

Comment:
Hum, thought it would be a way to introduce some tests on that class, but I can remove it